Makefile

```diff
@@ -15,7 +15,8 @@
 #  * PASSWORD: Desired password -- default at rules/config
 #  * KEEP_SLAVE_ON: Keeps slave container up after building-process concludes.
 #  * SOURCE_FOLDER: host path to be mount as /var/$(USER)/src, only effective when KEEP_SLAVE_ON=yes
-#  * SONIC_BUILD_JOB: Specifying number of concurrent build job(s) to run
+#  * SONIC_BUILD_JOBS: Specifying number of concurrent build job(s) to run
+#  * KERNEL_BUILD_METHOD: Specifying method of building kernel: download or build
```

Comment:
Same small concern I raised on the linked PR in sonic-linux-kernel: The terminology might not be intuitive (i.e., that one of the "build methods" is "download" which doesn't actually build anything). I think a name like "KERNEL_PROCURE_METHOD" is more accurate. Maybe someone else has another suggestion?